x-pack/plugins/timelines/public/components/t_grid/helpers.tsx

```diff
@@ -228,23 +228,22 @@ export const getCombinedFilterQuery = ({
   to,
   filters,
   ...combineQueriesParams
-}: CombineQueries & { from: string; to: string }): string => {
-  return replaceStatusField(
+}: CombineQueries & { from: string; to: string }): string =>
+  replaceStatusField(
     combineQueries({
       ...combineQueriesParams,
       filters: [...filters, buildTimeRangeFilter(from, to)],
-    })!.filterQuery
+    })?.filterQuery
   );
-};
 
 /**
  * This function is a temporary patch to prevent queries using old `signal.status` field.
  * @todo The `signal.status` field should not be queried anymore and
  * must be replaced by `ALERT_WORKFLOW_STATUS` field name constant
  * @deprecated
  */
-const replaceStatusField = (query: string): string =>
-  query.replaceAll('signal.status', ALERT_WORKFLOW_STATUS);
+const replaceStatusField = (filterQuery?: string): string =>
```

Comment:
I have a replacement for this in my UI PR, so don't worry about fixing it fully. My PR should take care of it

Reply:
Cool! this change is just to prevent the crash. I am happy to hear that this will be addressed properly 😊

Reply:
> I have a replacement for this in my UI PR, so don't worry about fixing it fully. My PR should take care of it

@michaelolo24 What's your PR?